server/drivers/linux_input.c

```diff
@@ -288,20 +286,35 @@ linuxInput_get_key (Driver *drvthis)
 	}
 
 	if (result != sizeof(event))
-		return NULL;
+		return -1;
 
 	/* Ignore release events and not-key events */
-	if (event.type != EV_KEY || event.value == 0)
+	return (event.type == EV_KEY && event.value) ? event.code : 0;
+}
+
+/**
+ * Helper function to convert a key code to a key name.
+ * \param p      Pointer to driver linuxInput PrivateData structure
+ * \param code   Linux KEY_ key-code to convert
+ * \retval       String representation of the key;
+ *               \c NULL for nothing available / error.
+ */
+const char *
+linuxInput_key_code_to_key_name (PrivateData *p, uint16_t code)
```

Comment:
Make this function static?

Reply:
Ack, good one, I will push a new version with this fixed in a minute.